src/main/java/net/snowflake/ingest/streaming/internal/FileLocationInfo.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+/** Class used to deserialized volume information response by server */
+class FileLocationInfo {
+  @JsonProperty("locationType")
+  private String locationType; // The stage type
```

Comment:
nit: can all the comments be made into regular javadoc comments with /* and */ ?

Reply:
Updated.